### PR TITLE
Remove unnecessary dependencies

### LINKED ISSUE #13: Unmet dependencies when building with crossbuilder
I am trying to build account-plugins using crossbuilder, as described in the [docs](https://docs.ubports.com/en/latest/appdev/system-software.html#cross-building-with-crossbuilder), getting an dependency issue:
```
builddeps:account-plugins:armhf : Depends: gobject-introspection:armhf but it is not going to be installed
                                   Depends: python3:armhf but it is not going to be installed
                                   Depends: libaccount-plugin-1.0-dev:armhf (>= 0.1.9) but it is not going to be installed
                                   Depends: valac:armhf (>= 0.16) but it is not going to be installed
                                   Depends: gnome-common:armhf but it is not installable
```
This may be related to ubports/crossbuilder#4.

### PR DESCRIPTION
**Pull request by jonnius**
Removed vala, gnome-common, python4, introspection dependencies and files. With this commit, crossbuilder can be used to build the packages. Fixes #13 

Should this go to xenial or master branch?

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -5,16 +5,11 @@ Maintainer: Ubuntu Desktop Team <ubuntu-desktop@lists.ubuntu.com>
...
                autotools-dev,
                intltool,
-               gobject-introspection,
-               python3,
                pkg-config,
-               libaccount-plugin-1.0-dev (>= 0.1.9),
                libaccounts-glib-dev (>= 1.10),
                libjson-glib-dev,
                libsignon-glib-dev,
                libsoup2.4-dev,
```
Comment (mardy) [inline debian/control]: Remove also the three above (starting from libjson-glib-dev)

**Review by mardy — COMMENTED** [review]
Thanks Jonnius, this looks excellent. Just please fix the debian/control as per my inline comment, and I'll merge this.
And yes, indeed xenial is the correct branch.

**Comment by jonnius** [comment]
I removed them.

**Comment by mardy** [comment]
Thanks! Can you please add one more commit, where you bring in this Jenkinsfile: https://github.com/ubports/build-tools/blob/master/Jenkinsfile ?
CI is failing, and I think that this could fix it.

**Comment by jonnius** [comment]
Unfortunately that didn't fix it. It just reenabled arm64, which has been disabled recently by @Flohack74 (#9, see the [commit](https://github.com/ubports/account-plugins/pull/9/commits/10c4a629e7f8856da66aef9d70f281901fa6847e)) due to jenkins failing to build that. Therefore, I removed the commit again...

**Comment by mardy** [comment]
Yes, I've seen that the arm64 build was disable, but I suspect that this is the cause why the later parts of the build are failing :-)
So, we should enable the arm64 build and try to find out why it fails, and fix it.

Do you have a log for the failed build? I can't seem to be able to find it in Jenkins.

**Comment by jonnius** [comment]
Weird, it still does not build for arm64...

**Comment by mardy** [comment]
@Flohack74 why did you merge this? We were still investigating why the build fails in CI :-)
Anyway, no need to revert: we can add the fix in another MP.

**Comment by mardy** [comment]
Actually, it looks like CI is working fine now, in the xenial branch :-)

**Comment by Flohack74** [comment]
@mardy Dont pannick, I was almost confident it will work - nah had no idea, but sometimes doing gives more chances to solve stuff than just pondering xD

**Pull request merged**